dashboard/templates/grades.html

```diff
@@ -19,7 +19,7 @@
     {% endif %}
 {% endblock %}
 {% block js %}
-    <script src="{% static 'd3-v4/build/d3.min.js' %}"></script>
+    <script src="{% static 'd3/dist/d3.min.js' %}"></script>
```

Comment:
The reason we used `yarn` is we are using 2 different versions of the `d3 library` and `yarn` supported to install 2 different version of same library. I don't think it would be a problem as the React seems to be using the latest3` version. I am thinking the existing viz might not load properly

Reply:
yes, I actually removed the d3@4 version since we're probably only need one version once the react rewrite is done. I don't think we're currently using vis in the rewrite yet, but if its a problem we can try importing a second version again

Reply:
Ya that's right, yarn [seemed the best way to have two versions of the same library available.](https://stackoverflow.com/questions/26414587/how-to-install-multiple-versions-of-package-using-npm)

Back when this was created we were using nvd3 for some charts and that only supported d3 v3. It looks like that still doesn't support anything newer. However we migrated away from that and some pages used d3 v4 but some just didn't get checked for upgrading.